docs/09-英文/01-Introduction/02quickstart.md

```diff
@@ -0,0 +1,93 @@
+# Quickstart
+
+This section will introduce the method of quickly building and deploying a single-Master RocketMQ cluster to complete simple message sending and receiving.
+
+## 1. Get Apache RocketMQ
+
+:::tip Download RocketMQ
+
+RocketMQ's installation is divided into two types: binary and source. Click [here](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-source-release.zip) to download Apache RocketMQ 4.9.4 source package, or download the binary package from [here](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-bin-release.zip). The binary package can be run directly since it has been compiled, and the source package needs to be compiled and run.
```

Comment:
Is there a way to define a global version number to avoid static version number (4.9.4) here?

Reply:
Great point, but this might need some work in the frontend.

Reply:
> Great point, but this might need some work in the frontend.

IMO a new issue could be created to achieve it. It's OK for the first version.